server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriodServicesImpl.java

```diff
@@ -158,4 +225,21 @@ private String constructEmailContent (UUID reviewPeriodId, String reviewPeriodNa
                 <a href="%s/feedback/reviews?period=%s">Click here</a> to review and approve reviewer assignments in the Check-Ins app.""".formatted(reviewPeriodName, webAddress, reviewPeriodId);
     }
 
+    private void createReviewRequest(ReviewPeriod period,
+                                     UUID creatorId,
+                                     UUID revieweeId,
+                                     UUID reviewerId,
+                                     UUID templateId,
+                                     LocalDate dueDate) {
+        try {
+            LocalDate sendDate = LocalDate.now();
+            FeedbackRequest request = new FeedbackRequest(
+                creatorId, revieweeId, reviewerId, templateId, sendDate,
+                dueDate.isAfter(sendDate) ? dueDate : sendDate.plusDays(1),
```

Comment:
```dueDate.isAfter(sendDate) ? dueDate : sendDate.plusDays(1),``` 
createReviewRequest ensures that the due date is always after the send date, which is not desirable. I recommend more robust coverage of this functionality to meet business requirements.

Reply:
@mkimberlin This is a good point.  Really, this shouldn't happen.  I think I ran into this with some pre-baked review periods.  Saving the feedback request when the send date is after the due date throws an exception.  Should we just not create feedback requests in this situation?

Reply:
Actually, this can definitely happen, in general.  The application allows review periods to be created with a close date in the past.  And, I am using the close date of the review period as the due date for feedback requests.  Maybe the fix is to disallow approval of review periods with a close date in the past?  Then, we just allow feedback requests to "fail" if internal feedback request validation fails and we log those errors.

Reply:
@ocielliottc Yeah, we should enforce sanity on those dates in the service calls that save and update review periods (and probably call it out as an error on the UI as well). I'll leave it to you if you want to do that in the context of another issue/PR or not. I would suggest the following:

 * If the review period is not yet "Open", then:
   * Launch Date must be in the future
 * In general: 
   * Self-Review Date must be after Launch Date
   * Close Date must be after Self-Review Date
   * Period Start Date must be before Launch Date
   * Period End Date must be after Period Start Date

Reply:
Excellent!  I'll do this in this PR.